system/ee/language/english/search_lang.php

```diff
@@ -152,6 +152,7 @@
 
     'today' => 'Today',
 
+    'expired_search_results' => 'The search have expired. Please, try again.',
```

Comment:
@matthewjohns0n need your input here as native English speaker :). How can we word this better? Also maybe worth removing "please try again" part completely, as some people will just try to refresh the page, which will not help

Reply:
@intoeetive I made a few tweaks to the message